spread/lxd.go

```diff
@@ -443,9 +478,15 @@ func (p *lxdProvider) address(name string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	for _, addr := range sjson.State.Network["eth0"].Addresses {
-		if addr.Family == "inet" && addr.Address != "" {
-			return addr.Address, nil
+	for ifacename, ifaceconf := range sjson.State.Network {
+		if ifacename == "lo" {
+			// ignore loopback interface
+			continue
+		}
+		for _, addr := range ifaceconf.Addresses {
+			if addr.Family == "inet" && addr.Address != "" {
+				return addr.Address, nil
+			}
 		}
```

Comment:
This one overlaps with #178, right? If this gets merged first, no need to merge #178.

Reply:
yup, I think I mentioned somewhere that I had exact same change :) I will rebase as needed